content/articles/data-access-with-ado-dotnet-in-micro-focus-visual-cobol/index.md

```diff
@@ -0,0 +1,217 @@
+### Introduction
+Micro Focus Visual COBOL is a modern development package that enables developers to maintain, create, and upgrade their systems for streamlining application development and delivery. Visual COBOL is a multi-language Integrated Development Environment (IDE).
+
+`ADO` is a Microsoft technology that stands for `ActiveX Data Object`. It is a collection of [ActiveX](https://en.wikipedia.org/wiki/ActiveX) controls that give you programmatic access to Microsoft's most recent data access technologies. As the name suggests, it's a wrapper for fundamental database access techniques.
+
+In simple words, ADO acts as a middleware between the programming language and the database. The developers write programs to access the data, without knowing how the database is implemented.
+
+`ADO.NET` is the next step in data access technology advancement. `ADO.NET` extends this by adding XML into a common model that applies to both relational and text-based XML data.
```

Comment:
``ADO.NET` extends this by adding XML into a common model that applies to both relational and text-based XML data.`

Could you explain more about this? This looks unclear